cms/djangoapps/contentstore/views/tests/test_transcripts.py

```diff
@@ -628,6 +629,86 @@ def test_check_youtube(self):
             }
         )
 
+    def mocked_requests_get(*args, **kwargs):
```

Comment:
didn't you just add the same function to https://github.com/edx/edx-platform/blob/rc/2015-06-09/cms/djangoapps/contentstore/tests/test_transcripts_utils.py#L324? Abstract it into a utility function to be used in both places